### PR TITLE
UCS/BITMAP: fixed typos

### PR DESCRIPTION
**Pull request by hoopoepg**

### CODE REVIEW

File: src/ucs/datastruct/bitmap.h [hunk 1]
```diff
@@ -90,7 +90,7 @@ _ucs_bitmap_word_index(size_t bitmap_words, size_t bit_index)
  * @param _bitmap    Take the word from this bitmap
  * @param _bit_index Index of the bit for fetching the word
  *
- * @return The word which containt
+ * @return The word which contains
```
Comment (gleon99) [inline src/ucs/datastruct/bitmap.h:93]: This sentence is weird (it was weird from the beginning). Maybe improve it?

Comment (hoopoepg) [reply src/ucs/datastruct/bitmap.h]: ok, updated

**Comment by gleon99** [comment]
👍

**Review by gleon99 — APPROVED** [review]

**Review by brminich — APPROVED** [review]

**Comment by hoopoepg** [comment]
@yosefe ok to merge?

**Pull request merged**